### PR TITLE
fixed lat lon inversion in rosbag waypoint parsing, added paths from mock rolls

### PR DESCRIPTION
**Pull request by Jackack**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
See title

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

- [ ] Yes
- [x] No, and this is why: tested rosbag to waypoint script irl. No tests required for all the paths
- [ ] I need help with writing tests

### CODE REVIEW

File: rb_ws/src/buggy/paths/frew_test_mar21_2.json
Comment (mehulgoel873) [inline rb_ws/src/buggy/paths/frew_test_mar21_2.json:1]: Fix or delete this file name.

File: rb_ws/src/buggy/paths/outside_curb_clean.json
Comment (mehulgoel873) [inline rb_ws/src/buggy/paths/outside_curb_clean.json:1]: Rename file

File: rb_ws/src/buggy/paths/outside_curb_smooth.json
Comment (mehulgoel873) [inline rb_ws/src/buggy/paths/outside_curb_smooth.json:1]: Rename file

**Comment by christianvluu** [comment]
just make sure the curbs are reversed as necessary

**Review by TiaSinghania — APPROVED** [review]

**Review by TiaSinghania — APPROVED** [review]

**Pull request merged**